backend/organization/views/project_views.py

```diff
@@ -942,17 +937,14 @@ class IsUserFollowing(APIView):
     permission_classes = [IsAuthenticated]
 
     def get(self, request, url_slug):
-        try:
-            project = Project.objects.get(url_slug=url_slug)
-        except Project.DoesNotExist:
-            raise NotFound(
-                detail="Project not found:" + url_slug, code=status.HTTP_404_NOT_FOUND
-            )
-
-        is_following = ProjectFollower.objects.filter(
-            user=request.user, project=project
-        ).exists()
-        return Response({"is_following": is_following}, status=status.HTTP_200_OK)
+        return check_if_user_follows(
+            request.user,
+            url_slug,
+            Project,
+            ProjectFollower,
+            "project",
+            "Project not found",
+        )
```

Comment:
Good job to make these generic :+1: Potentially we could even create a `check_if_user_follows_project` function which then calls `check_if_user_follows`. This would hide even more complexity

Reply:
Hid complexity for all functions in `organzation/utility/follow.py` with helper functions:
- check_if_user_follows_project
- check_if_user_follows_organization
- set_user_following_project
- set_user_following_organization
- get_list_of_organization_followers
- -get_list_of_project_followers